### PR TITLE
add ScanDetailV2 in GetResponse and BatchGetResponse

### PR DESCRIPTION
**Pull request by Patrick0308**
See https://github.com/tikv/tikv/issues/8756.

This pr add add ScanDetailV2 in GetResponse and BatchGetResponse.

### CODE REVIEW

**Review by zz-jason — COMMENTED** [review]
please run `make` to generate the library and commit them to this PR.

**Review by breeswish — COMMENTED** [review]
LGTM, thanks! Please run `make` to generate corresponding Protobuf file.

**Comment by Patrick0308** [comment]
@breeswish @zz-jason I found a problem. Please see #683

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -36,6 +36,8 @@ message GetResponse {
     bytes value = 3;
     // True if the key does not exist in the database.
     bool not_found = 4;
+    // set when ctx.scan_detail = true or meet slow query
```
Comment (wjhuang2016) [inline proto/kvrpcpb.proto:39]: Set when ctx.scan_detail = true or meet slow query.

File: proto/kvrpcpb.proto [hunk 2]
```diff
@@ -304,6 +306,8 @@ message BatchGetRequest {
 message BatchGetResponse {
     errorpb.Error region_error = 1;
     repeated KvPair pairs = 2;
+    // set when ctx.scan_detail = true or meet slow query
```
Comment (wjhuang2016) [inline proto/kvrpcpb.proto:309]: ditto.

**Review by wjhuang2016 — COMMENTED** [review]
LGTM

**Review by breeswish — APPROVED** [review]

**Comment by breeswish** [comment]
LGTM, thanks!

**Comment by Patrick0308** [comment]
@wjhuang2016 Please approve it.

**Review by wjhuang2016 — APPROVED** [review]

**Pull request merged**